armv7 version crashes with panic: unaligned 64-bit atomic operation

When running livekit on armv7 (32bit) livekit crashes as soon as a client connects to the server

**Server**
- Version: [1.5.2]
- Environment: local dev, debian experimental armv7

**Client**
- SDK: element call livekit branch (uses livekit js 0.8.2 as far as I know)
- Version: [0.8.2]

**To Reproduce**
join element call call

**Expected behavior**
simply working


**Additional context**
```
panic: unaligned 64-bit atomic operation

goroutine 39 [running]:
runtime/internal/atomic.panicUnaligned()
        /opt/hostedtoolcache/go/1.21.5/x64/src/runtime/internal/atomic/unaligned.go:8 +0x24
runtime/internal/atomic.Xadd64(0x26f51cc, 0x1)
        /opt/hostedtoolcache/go/1.21.5/x64/src/runtime/internal/atomic/atomic_arm.s:258 +0x14
github.com/ua-parser/uap-go/uaparser.(*Parser).ParseOs(0x26f5180, {0x2442050, 0x46})
        /home/runner/go/pkg/mod/github.com/ua-parser/uap-go@v0.0.0-20230823213814-f77b3e91e9dc/uaparser/parser.go:318 +0x17c
github.com/ua-parser/uap-go/uaparser.(*Parser).Parse.func2()
        /home/runner/go/pkg/mod/github.com/ua-parser/uap-go@v0.0.0-20230823213814-f77b3e91e9dc/uaparser/parser.go:249 +0xa0
created by github.com/ua-parser/uap-go/uaparser.(*Parser).Parse in goroutine 36
        /home/runner/go/pkg/mod/github.com/ua-parser/uap-go@v0.0.0-20230823213814-f77b3e91e9dc/uaparser/parser.go:246 +0x1ac
```

I tried a somewhat older version. that one crashed on a different place (probably fixed already), but since I don't see many issues with arm 32 bit I mentioning straight crashes I think nobody really uses arm32? If that is the case and you don't want to waste time with that kind of issues I have no problem with that, but in that case I would suggest removing the arm32 bit build to make everyone aware that only 64bit archs are expected to work/supported

--- a/pkg/rtc/participant.go
+++ b/pkg/rtc/participant.go
@@ -135,6 +135,10 @@ type ParticipantParams struct {
 }
 
 type ParticipantImpl struct {
+	// utils.TimedVersion is a atomic. To be correctly aligned also on 32bit archs
+	// 64it atomics need to be at the front of a struct
+	timedVersion utils.TimedVersion
+
 	params ParticipantParams
 
 	isClosed    atomic.Bool
@@ -195,7 +199,6 @@ type ParticipantImpl struct {
 
 	dirty        atomic.Bool
 	version      atomic.Uint32
-	timedVersion utils.TimedVersion
 
 	// callbacks & handlers
 	onTrackPublished     func(types.LocalParticipant, types.MediaTrack)

--- a/pkg/rtc/room.go
+++ b/pkg/rtc/room.go
@@ -67,6 +67,15 @@ type broadcastOptions struct {
 }
 
 type Room struct {
+	// atomics always need to be 64bit/8byte aligned
+	// on 32bit arch only the beginning of the struct
+	// starts at such a boundary.
+	// time the first participant joined the room
+	joinedAt atomic.Int64
+	// time that the last participant left the room
+	leftAt atomic.Int64
+	holds    atomic.Int32
+
 	lock sync.RWMutex
 
 	protoRoom  *livekit.Room
@@ -96,11 +105,6 @@ type Room struct {
 	batchedUpdates   map[livekit.ParticipantIdentity]*livekit.ParticipantInfo
 	batchedUpdatesMu sync.Mutex
 
-	// time the first participant joined the room
-	joinedAt atomic.Int64
-	holds    atomic.Int32
-	// time that the last participant left the room
-	leftAt atomic.Int64
 	closed chan struct{}
 
 	trailer []byte

--- a/pkg/rtc/uptrackmanager.go
+++ b/pkg/rtc/uptrackmanager.go
@@ -37,6 +37,10 @@ type UpTrackManagerParams struct {
 
 // UpTrackManager manages all uptracks from a participant
 type UpTrackManager struct {
+	// utils.TimedVersion is a atomic. To be correctly aligned also on 32bit archs
+	// 64it atomics need to be at the front of a struct
+	subscriptionPermissionVersion utils.TimedVersion
+
 	params UpTrackManagerParams
 
 	closed bool
@@ -44,7 +48,6 @@ type UpTrackManager struct {
 	// publishedTracks that participant is publishing
 	publishedTracks               map[livekit.TrackID]types.MediaTrack
 	subscriptionPermission        *livekit.SubscriptionPermission
-	subscriptionPermissionVersion utils.TimedVersion
 	// subscriber permission for published tracks
 	subscriberPermissions map[livekit.ParticipantIdentity]*livekit.TrackPermission // subscriberIdentity => *livekit.TrackPermission
 

